modules/currency.js

```diff
@@ -11,6 +11,7 @@ const CURRENCY_RATE_PRECISION = 4;
 var bidResponseQueue = [];
 var conversionCache = {};
 var currencyRatesLoaded = false;
+var needToCallForCurrencyFile = true;
```

Comment:
global state is a bad practice and should be refactored but I won't block the PR as it's already all over this module...